pom.xml

```diff
@@ -584,6 +589,12 @@
         <groupId>com.google.api</groupId>
         <artifactId>gax-grpc</artifactId>
         <version>${gax-grpc.version}</version>
+        <exclusions>
+          <exclusion>
+            <groupId>com.google.api.grpc</groupId>
+            <artifactId>proto-google-common-protos</artifactId>
```

Comment:
Done, but please see https://github.com/googleapis/toolkit/pull/1382 for context.